### PR TITLE
Add ADR for archiving of the DNS repo

### PR DESCRIPTION
**Pull request by ben-al**

### CODE REVIEW

**Review by andyrogers1973 — DISMISSED** [review]
lgtm

File: source/documentation/adrs/adr-013.html.erb.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Context
+
+The [DNS repo](https://github.com/ministryofjustice/dns) has historically held the following:
+
+- A dump of the dsd.io and justice.gov.uk hosted zones in Route53
+- Tooling to register domains on bulk
+- Tooling to maintain defensive domains
+
+The repository has not been in use for a long time now and is very out of date.
```
Comment (tamsinforbes) [inline source/documentation/adrs/adr-013.html.erb.md:22]: Is it useful to say how long roughly? Eg "for 6-9 months" or since <date>.

**Review by tamsinforbes — APPROVED** [review]
Left a comment re time since last used.

File: source/documentation/adrs/adr-013.html.erb.md (hunk 1 above)
Comment (ben-al) [reply source/documentation/adrs/adr-013.html.erb.md]: Will add it!

**Review by andyrogers1973 — APPROVED** [review]
lgtm

**Pull request merged**